src/main/features/claim/views/eligibility/not-eligible.njk

```diff
@@ -7,6 +7,7 @@
 {% block content %}
 
   {% set n1FormUrl = 'https://formfinder.hmctsformfinder.justice.gov.uk/n1-eng.pdf' %}
+  {% set moneyClaimOnline = 'https://www.moneyclaim.gov.uk/web/mcol/welcome' %}
```

Comment:
What the `moneyClaimOnline` is? Name is not meaningful enough for me. What about `legacyServiceUrl`

Reply:
In that sense, the n1FormUrl isnt meaningful enough either. 
I think legacyServiceUrl is a good recommendation. Ill add now

Reply:
huh, how is it not meaningful?
I would be fine with leaving how it is.

Reply:
Unless you know exactly what the N1 form does, its no more helpful than a "m84FormUrl"